lib/rsa/rsa-verify.c

```diff
@@ -380,6 +384,11 @@ static int rsa_verify_with_keynode(struct image_sign_info *info,
 	if (!prop.public_exponent || length < sizeof(uint64_t))
 		prop.public_exponent = NULL;
 
+#if defined(CONFIG_SPL_RSA)
```

Comment:
Let's get some more testing on this code to verify that we need this fix.

Reply:
that patch for aligment work around is still needed, otherwise:

```
   Trying to boot from MMC2
    sha256,rsa2048:dev"Synchronous Abort" handler, esr 0x96000021
    elr: 00000000007e737c lr : 00000000007e6e78
    x 0: 00000000007fd1e4 x 1: 0000000000000100
    x 2: 0000000000000800 x 3: 000000000091d420
    x 4: 000000000091d420 x 5: 000000000091d588
    x 6: 00000000401ff9b8 x 7: 0000000000000003
    x 8: 000000000091d49c x 9: 0000000000000002
    x10: 00000000007fcec8 x11: 00000000307c6ebb
    x12: 000000000d058cc7 x13: 00000000dfd68b5f
    x14: 0000000084328582 x15: 0000000077680b62
    x16: 00000000007f1cd4 x17: 0000000050808008
    x18: 000000000091dea0 x19: 000000000091d520
    x20: 0000000000000100 x21: 000000000091d420
    x22: 00000000401ff9b8 x23: 000000000091d5b0
    x24: 0000000000000100 x25: 00000000007fad58
    x26: 0000000000000020 x27: 000000000091da60
    x28: 0000000000000008 x29: 000000000091d3a0
    Code: f94008a0 2a0103f4 aa0303f5 b40005c0 (f9400003)
    Resetting CPU ...
```